src/HydrationRepository.js

```diff
@@ -5,10 +5,10 @@ class HydrationRepository {
     this.hydrationData = this.instantiateHydroData(rawHydrationData);
   }
 
-  instantiateHydroData(rawHydrationData) {
```

Comment:
What was the motivation to instantiate each hydration data when no methods or properties are added to it? Is this redundant?